emcc.py

```diff
@@ -2320,6 +2319,24 @@ def do_binaryen(final, target, asm_target, options, memfile, wasm_binary_target,
           passes.append('minifyWhitespace')
         final = shared.Building.js_optimizer_no_asmjs(final, passes)
       if DEBUG: save_intermediate('postclean', 'js')
+  # replace placeholder strings with correct subresource locations
+  if shared.Settings.SINGLE_FILE:
+    f = open(final, 'r')
+    js = f.read()
+    f.close()
+    f = open(final, 'w')
+    for tup in [
+      (wasm_text_target, shared.FilenameReplacementStrings.WASM_TEXT_FILE, True),
+      (wasm_binary_target, shared.FilenameReplacementStrings.WASM_BINARY_FILE, True),
+      (asm_target, shared.FilenameReplacementStrings.ASMJS_CODE_FILE, not shared.Building.is_wasm_only())
```

Comment:
Looks like a force-push removed this commit, so I'm not sure how to see context for this question. Is it still relevant?